setup.py

```diff
@@ -172,6 +170,12 @@ def generate_build_ext(pkg_dir: Path, pkg_name: str):
             "-O3",
             "-fvisibility=hidden",
         ]
+        # add c++20 or c++2a flag depending on gcc version
+        # TODO remove this if musllinux upgrade to newer gcc version
+        if "GCC 9." in sys.version:
+            cflags += ["-std=c++2a"]
```

Comment:
how important is it that we support musllinux? this strongly restricts us in the specific C++20 features and library features we can use. e.g. the spaceship `<=>` operator is not supported until GCC 10

Reply:
@SteffanWullems wants this in his team because he would like to use alpine based image. I can understand that. Since numpy 1.25 now supports musl, I would also go for it for PGM. I suppose `musllinux_1_2` should have more receng GCC version. You can try to contribute a `musllinux_1_2` image. See https://github.com/pypa/manylinux/issues/1488.

Reply:
ok if there are known users requesting it then that is a good reason :+1: